native/ilaenv.go

```diff
@@ -5,10 +5,11 @@
 package native
 
 // Ilaenv returns algorithm tuning parameters for the algorithm given by the
-// input string. ispec specifies the parameter to return.
-//  1: The optimal block size
-//  2: The minimum block size for which the algorithm should be used.
-//  3: The crossover point below which an unblocked routine should be used.
+// input string. ispec specifies the parameter to return:
+//  1: The optimal block size for a blocked algorithm.
+//  2: The minimum block size for a blocked algorithm.
+//  3: The block size of unprocessed data at which a blocked algorithm should
```

Comment:
This seems fine. Maybe change "switch" to "crossover" since that have a specific use in this context.